web/templates/web/garden/scientists.html

```diff
@@ -10,4 +10,17 @@
 {% block content %}
     {% url "web:lab-studies-list" lab_slug="garden" as garden_url %}
     {% include "web/garden/_navigation.html" %}
+    {% trans "Meet the Scientists" as trans_garden_scientists %}
+    {% page_title trans_garden_scientists %}
+    <p>
+        Project GARDEN would not be possible without the work of many scientists and collaborators! Meet the team working hard to help families contribute to developmental science.
+    </p>
+    <h2 class="{% subheading_classes %}">Leadership Team</h2>
+    {% include "web/garden/_leadership_team.html" %}
+    <h2 class="{% subheading_classes %}">Grant Team Members</h2>
+    {% include "web/garden/_grant_team.html" %}
+    <h2 class="{% subheading_classes %}">Research Partners</h2>
+    {% include "web/garden/_research_partners.html" %}
+    <h2 class="{% subheading_classes %}">Other team members</h2>
+    {% include "web/garden/_other_members.html" %}
```

Comment:
If `subheading_classes` is just a list of BS5 classes, then you could create a new class instead.

Reply:
Yes, will do!
BTW I think there are a few other template tags that are just simple lists of classes. I'll open a separate issue/PR to change those too.